### ISSUE TITLE
Issue of drmaa-python not directing the standard error to my desired filepath

### ISSUE THREAD

**Issue by kohaugustine** [pygridtools/drmaa-python#20]
Dear All DRMAA Python Users and Developers,

I have the abovementioned issue, and have not been able to direct the standard error messages out to the desired file path whenever I run a process on my distributed SGE-controlled cluster. In my Python code, I have the following statements that are:

    jt.outputPath = ':' + self.stdout_log_filepath  
    jt.errorPath = ':' + self.stderr_log_filepath

And in my debugging I have inserted print statements before and after the `errorPath` line to ensure that it was actually executed. I have run the same process on a local setting and been able to get both the standard error and standard output be written to my desired files. With further debugging, I examined the contents of the `self.stdout_log_filepath` standard output when I ran it in a distributed manner, and realized that what I saw in `self.stderr_log_filepath` when I ran the process locally was actually found in the `self.stdout_log_filepath` standard output file in the distributed case. 

Somehow it seems like DRMAA has not separated the standard error from the standard output streams, and the standard error stream gets written out to the same output file path as the standard error, and thus both std error and std output both go into `self.stdout_log_filepath`. But I really need the standard error to be separated and to go into its `self.stderr_log_filepath` file. 

Could this be a bug? Or am I using the API wrongly? I would greatly appreciate any help on this issue to solve it. 

Thank you very much!

**Comment by dan-blanchard** [comment]
There are three reasons I can think of that that would happen:

1.  `jt.joinFiles` is `True`
2.  You've specified `-j yes` in `jt.nativeSpecification`
3.  Your SGE cluster has `-j yes` set by default.  (This is a pretty common setting.)

I would suggest trying to manually set `jt.joinFiles` to `False` and see if that helps (and making sure that you're not setting `-j yes` in `jt.nativeSpecification`).

**Comment by kohaugustine** [comment]
@dan-blanchard  Thank you very much for your quick response. Yes I've indeed identified the problem to be number 1, that I specified `jt.joinFiles = True` .  Now I've set it to `False`, and the problem is solved. When I checked back to the documentation, I couldn't find an explanation on this attribute in the Python-Drmaa documentation, but was able to find it in the original DRMAA API Specification. It'll be good if this attribute of `jt` can also be explained in the python version, as the tutorial uses it in the examples, but does not explain it.
